core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.ts

```diff
@@ -419,7 +421,8 @@ export class TopicsAndSkillsDashboardPageComponent {
           this.initSkillDashboard();
           this.focusManagerService.setFocus('createSkillBtn');
         }
-        this.classrooms = response.allClassroomNames;
+
+        this.classrooms.push(...response.allClassroomNames);
```

Comment:
Why this change? Looks strange

Reply:
we have a default value as "All Classrooms" which return topics which are assigned to some classroom and we add rest of the available classrooms to filter later.